examples/scripts/reward_trainer.py

```diff
@@ -63,6 +64,7 @@ class ScriptArguments:
 
 parser = HfArgumentParser(ScriptArguments)
 script_args = parser.parse_args_into_dataclasses()[0]
+print(script_args)
```

Comment:
```suggestion
```

Reply:
Haha nice catch, `print()` is the best debugger :D